### PR TITLE
Backend: add missing unit test for ModuleService class

### PR DESCRIPTION
**Pull request by dmytro-ch**
### Description
This PR adds missing unit tests for Module List Service class:
- `\Magento\Backend\Service\V1\ModuleService`

### Fixed Issues (if relevant)
N/A

### Manual testing scenarios
N/A

### Contribution checklist
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: app/code/Magento/Backend/Test/Unit/Service/V1/ModuleServiceTest.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Backend\Test\Unit\Model;
```
Comment (rogyar) [inline app/code/Magento/Backend/Test/Unit/Service/V1/ModuleServiceTest.php]: Please, adjust the namespace to correspond the file path

**Review by rogyar — CHANGES_REQUESTED** [review]

**Comment by ishakhsuvarov** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @ishakhsuvarov. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @ishakhsuvarov, here is your [new Magento instance](http://ec2-34-228-235-121.compute-1.amazonaws.com/pr-18546/).
Admin access: [http://ec2-34-228-235-121.compute-1.amazonaws.com/pr-18546//admin](http://ec2-34-228-235-121.compute-1.amazonaws.com/pr-18546//admin)
Login: `admin` Password: `123123q`

File: app/code/Magento/Backend/Test/Unit/Service/V1/ModuleServiceTest.php (hunk 1 above)
Comment (dmytro-ch) [reply app/code/Magento/Backend/Test/Unit/Service/V1/ModuleServiceTest.php]: @rogyar thank you!
I've updated  the namespace.

**Review by rogyar — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @rogyar, thank you for the review.
[ENGCOM-3167](https://jira.corp.magento.com/browse/ENGCOM-3167) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch. Thank you for your contribution. 
We will aim to release these changes as part of 2.3.1. 
Please check the release notes for final confirmation.

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch. Thank you for your contribution. 
We will aim to release these changes as part of 2.3.1. 
Please check the release notes for final confirmation.

**Pull request merged**